### DISCUSSION TITLE
I'd be cool to see more settings depending on usage or battery life

### DISCUSSION THREAD

**Discussion by MusicalArtist12** [AdnanHodzic/auto-cpufreq#616] [Ideas]
currently, `auto-cpufreq.conf` only has two modes: `[charger]` and `[battery]`. i'd be cool to see something like: 
```
[battery-100]
# active when the battery is below 100%

[battery-50]
# active when the battery falls below 50%

[battery-10]
# active when the battery falls below 10%
```

**Comment by AdnanHodzic** [comment]
Agreed, I would turn this into a "[Feature request]" issue. Especially since there are existing issues (can't find them now) requesting same functionality.
